README.md

```diff
@@ -1,7 +1,30 @@
-# Diff en Rust
+# Diff in Rust
 
-Un programa para encontrar las diferencia entre dos archivos.
+A small program to find the differences (line by line) between two files.
 
-## Runing the program
+## Running the program
 
 `cargo run file1 file2`
+
+## A small example
+
+file1 contents:
+```
+Hello,
+World!
+```
+
+file2 contents:
+```
+Good Morning,
+World!
+```
+
+When `cargo run file1 file1` is executed, the output is:
```

Comment:
It should say `cargo run file1 file2`

Reply:
Corrected in commit d68b7cbaa0fba10234bdb52a38e368230d64f42b